_includes/pages/index/get-kotlin.html

```diff
@@ -15,7 +15,7 @@ <h3 class="option-title">Compiler</h3>
                 </p>
 
                 <div class="option-link-wrap">
-                    <a href="{{ site.baseurl }}/docs/tutorials/getting-started.html"
+                    <a href="{{ site.baseurl }}/docs/tutorials/command-line.html"
                        class="option-link">Download Compiler</a>
```

Comment:
I think we wanted to point to the tutorial. The link to the compiler itself is the first thing in the tutorial.